secondary-server-1/src/main/java/com/replicated_log/secondary_server_1/config/InitialDataConfiguration.java

```diff
@@ -0,0 +1,32 @@
+package com.replicated_log.secondary_server_1.config;
+
+import com.replicated_log.secondary_server_1.model.Address;
+import com.replicated_log.secondary_server_1.service.SecondaryClient;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+import javax.annotation.PostConstruct;
+
+@Configuration
+public class InitialDataConfiguration {
+
+    private final Logger LOG = LogManager.getLogger(InitialDataConfiguration.class);
+
+    @Value("${secondary.server.baseurl}")
+    private String SECONDARY_SERVER_BASE_URL;
+
+    @Autowired
+    private SecondaryClient secondaryClient;
+
+    @PostConstruct
+    public void postConstruct() {
```

Comment:
Configuration class should no handle any business logic, it is just setup of your app. If you need do something after start app try to use spring listeners or command line runners

Reply:
used @EventListener(ApplicationReadyEvent.class) to send Address of Secondary to Master after start app